README.md

```diff
@@ -1,204 +1,7 @@
-[![Build status](http://ci.virtocommerce.com/buildStatus/icon?job=vc-2-org/vc-theme-default/master)](http://ci.virtocommerce.com/job/vc-2-org/job/vc-theme-default/job/master/) [![Quality Gate](https://sonar.virtocommerce.com/api/badges/gate?key=vc-theme-default%3Amaster&blinking=true)](https://sonar.virtocommerce.com/dashboard?id=vc-theme-default%3Amaster) [![Lines of code](https://sonar.virtocommerce.com/api/badges/measure?key=vc-theme-default%3Amaster&metric=ncloc)](https://sonar.virtocommerce.com/api/badges/measure?key=vc-theme-default%3Amaster&metric=ncloc)&emsp;
-[![Documentation](https://img.shields.io/badge/docs-read-brightgreen.svg)](https://virtocommerce.com/docs/vc2devguide/working-with-storefront/theme-development)&nbsp;[![Ask question at https://stackoverflow.com/questions/tagged/virtocommerce](https://img.shields.io/badge/stackoverflow-ask%20question-orange.svg)](https://stackoverflow.com/questions/tagged/virtocommerce) [![Join the chat at https://gitter.im/VirtoCommerce/vc-community](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/VirtoCommerce/vc-community?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge) [![Contributors](https://img.shields.io/github/contributors/VirtoCommerce/vc-theme-default.svg)](https://github.com/VirtoCommerce/vc-theme-default/graphs/contributors)
+Samples for [vc-theme-default](https://github.com/VirtoCommerce/vc-theme-default)
+========================
 
-_Default theme_ for VirtoCommerce Storefront used by _Electronics_ sample store. It includes the largest number of features and support latest updates of Storefront so it always stay actual.
+**Getting started** and all other related documentation can be found in the original repository https://github.com/VirtoCommerce/vc-theme-default.
```

Comment:
**Source code getting started** and all other related documentation can be found in the original repository https://github.com/VirtoCommerce/vc-theme-default.

Reply:
**Getting started** is the exact paragraph naming